ci-operator/step-registry/mco/confd2/kerneltype/mco-confd2-kerneltype-ref.yaml

```diff
@@ -0,0 +1,30 @@
+ref:
+  as: mco-confd2-kerneltype
+  from: cli
+  commands: mco-confd2-kerneltype-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+    - name: MCO_CONFD2_INSTALL_KERNEL_MCPS
```

Comment:
I think we can use the same var name as day0 operation.

Reply:
the orig design is use same var name with day1 step. after discussion within team, we think it's better to put explicit symbols in day2 vars. so it's very clear for user when they declare the env vars in chain/workflow

Reply:
I have modified the var names to be like this: MCO_CONF_DAY1_INSTALL_KERNEL_MCPS and MCO_CONF_DAY2_INSTALL_KERNEL_MCPS

Two different steps using the same var names could collide, I would like to use different variable names in different steps if possible.